WalletWasabi.Gui/Global.cs

```diff
@@ -102,12 +102,11 @@ public Global(string dataDir, string torLogsFile, Config config, UiConfig uiConf
 				WalletManager.OnDequeue += WalletManager_OnDequeue;
 				WalletManager.WalletRelevantTransactionProcessed += WalletManager_WalletRelevantTransactionProcessed;
 
-				var indexStore = new IndexStore(Network, new SmartHeaderChain());
+				var networkWorkFolderPath = Path.Combine(DataDir, "BitcoinStore", Network.ToString());
+				var transactionStore = new AllTransactionStore(networkWorkFolderPath, Network);
+				var indexStore = new IndexStore(Path.Combine(networkWorkFolderPath, "IndexStore"), Network, new SmartHeaderChain());
 
-				BitcoinStore = new BitcoinStore(
-					Path.Combine(DataDir, "BitcoinStore"), Network,
-					indexStore, new AllTransactionStore(Network), new MempoolService()
-				);
+				BitcoinStore = new BitcoinStore(indexStore, transactionStore, new MempoolService());
```

Comment:
This is perfectly okay of course but I would prefer a coherent style:

* Do not use temporary variables and create the all the instances here or,
* Always use temporary variables and never create any instance in here.

Reply:
I fixed that.